### PR TITLE
[doc,site] Site/landing changes

### PR DESCRIPTION
**Pull request by hcallahan-lowrisc**
~~[Only consider the final 2 commits, this PR will be rebased once #17323 is merged,  which makes up all but the last 2 commits here.]~~

REVIEW BUT DO NOT MERGE
_We want to coordinate the merge of new website changes, so that suitable precautions can be taken for a safe deployment._

---

This PR consists of changes to the landing page site (opentitan.org) that were worked on out-of-tree by a number of people. The primary work was to change the site-header and add new pages in preparation for upcoming further features.
There has also been some minor refactoring to the existing code, and some styling tweaks.

---

Known Issues
- Mobile experience has issues which are yet to be addressed.

Screenshot from 2023-02-28 14-17-21

Screenshot from 2023-02-28 14-18-20

### CODE REVIEW

**Comment by hcallahan-lowrisc** [comment]
> * Mobile experience has issues which are yet to be addressed.

@marnovandermaas If you have any time available, would you mind checking this out? I think the wrap behaviour just needs looking at, or even going full hamburger :hamburger:

**Comment by hcallahan-lowrisc** [comment]
@jwnrt I've included most of the changes to the landing page site here from the refactor repo (minus the stats, which can come back later). Would you mind checking out if anything made it in that is questionable / needs further work? I know this is kind of a unfocused task/ask, but any ideas you have would be helpful. Cheers

**Comment by marnovandermaas** [comment]
> > * Mobile experience has issues which are yet to be addressed.
> 
> @marnovandermaas If you have any time available, would you mind checking this out? I think the wrap behaviour just needs looking at, or even going full hamburger hamburger

An initial implementation here: https://github.com/marnovandermaas/opentitan/tree/site/landing_changes
Couple of issues with it:
- Hamburger menu jumping to the left when going to smaller screens
- Animation of hamburger menu should be less fancy
- The menu itself does not respond to the light/dark theme

**Comment by hcallahan-lowrisc** [comment]
> > > * Mobile experience has issues which are yet to be addressed.
> > 
> > 
> > @marnovandermaas If you have any time available, would you mind checking this out? I think the wrap behaviour just needs looking at, or even going full hamburger hamburger
> 
> An initial implementation here: https://github.com/marnovandermaas/opentitan/tree/site/landing_changes Couple of issues with it:
> 
> * Hamburger menu jumping to the left when going to smaller screens
> * Animation of hamburger menu should be less fancy
> * The menu itself does not respond to the light/dark theme

Thanks for starting on this @marnovandermaas, it's much appreciated, and should be a good thing to build on. It's going to be nice to have a consistent mobile experience between both the landing-site and the book. :+1:

**Comment by hcallahan-lowrisc** [comment]
rebased

**Comment by jwnrt** [comment]
Looks good, thanks for putting all this together

Only issue I can see is that we've moved the skyline without moving the logo so it's not aligned any more

Should we leave out the skyline/logo change from this PR? I think it should be fine, we can add it later when we want to fit those metrics in this section

File: site/landing/content/privacy-policy.md [hunk 1]
```diff
@@ -1,8 +1,4 @@
----
-title: "Website Privacy and Cookies Policy"
-aliases:
-  - /privacy/
----
```
Comment (jwnrt) [inline site/landing/content/privacy-policy.md:5]: These pages are still Hugo, is it okay to remove the frontmatter? I think the `title:` is inserted into the browser tab title

**Comment by hcallahan-lowrisc** [comment]
> Should we leave out the skyline/logo change from this PR? I think it should be fine, we can add it later when we want to fit those metrics in this section

Whoops yes that should come later. Is that just a case of removing the changes to the static .svg files, or is there other things needed too?

File: site/landing/content/privacy-policy.md (hunk 1 above)
Comment (hcallahan-lowrisc) [reply site/landing/content/privacy-policy.md]: Yep good point, I've just push a change re-adding the `title` frontmatter for all the static pages.

Comment (hcallahan-lowrisc) [reply site/landing/content/privacy-policy.md]: Testing locally, this is indeed needed for a meaningful message in the tab title.

**Comment by jwnrt** [comment]
> Is that just a case of removing the changes to the static .svg files, or is there other things needed too?

I think that should be fine, but I can see some other changes to margins etc. too, so not too sure.

I have a branch which adds these changes in a way that keeps all the CSS separate, would you mind if I submitted that as a PR?
@hcallahan-lowrisc

**Comment by hcallahan-lowrisc** [comment]
> > Is that just a case of removing the changes to the static .svg files, or is there other things needed too?
> 
> I think that should be fine, but I can see some other changes to margins etc. too, so not too sure.
> 
> I've made a branch with these changes split into separate CSS files, let me know what you think #17462

Sorry I'm confused, is that PR to be made on-top of this PR? or instead of it?

**Comment by hcallahan-lowrisc** [comment]
rebased

**Comment by hcallahan-lowrisc** [comment]
Closing in favour of #17462

**Pull request closed without merging**